components/wordPrompt.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+    const changeWordHandler = () => {
```

Comment:
The indentation seems a bit off here, `useState` and `fetchWords` seem to be within `changeWordHandler` if when they aren't